### PR TITLE
Update BambooTracker.desktop

### PR DESCRIPTION
**Pull request by trebmuh**
adds French comment and Generic name, and remove the final point (consistency with others desktop files)

### CODE REVIEW

File: BambooTracker.desktop [hunk 1]
```diff
@@ -2,9 +2,11 @@
 Name=BambooTracker
 GenericName=OPNA Tracker
 GenericName[jp]=OPNAのトラッカーです
-Comment=A tracker for the YM2608 (OPNA), which was used in NEC PC-8801/9801 series of computers.
+GenericName=Tracker OPNA
```
Comment (OPNA2608) [inline BambooTracker.desktop]: I think you forgot the locale specification
```suggestion
 GenericName[fr]=Tracker OPNA
```

File: BambooTracker.desktop [hunk 2]
```diff
@@ -2,9 +2,11 @@
...
+Comment=A tracker for the YM2608 (OPNA), which was used in NEC PC-8801/9801 series of computers
 Comment[jp]=このアプリケーションはNEC PC-8801/9801シリーズに搭載されていたFM音源YM2608(OPNA)向けのトラッカーです。
-Comment[de]=Ein Tracker für den YM2608 (OPNA), welcher in NECs PC-8801/9801 Serien an Computern verwendet wurde.
+Comment[de]=Ein Tracker für den YM2608 (OPNA), welcher in NECs PC-8801/9801 Serien an Computern verwendet wurde
+Comment[de]=Un tracker pour le YM2608 (OPNA), qui fut utilisé dans la séries d'ordinateurs NECs PC-8801/9801
```
Comment (OPNA2608) [inline BambooTracker.desktop]:
```suggestion
Comment[fr]=Un tracker pour le YM2608 (OPNA), qui fut utilisé dans la séries d'ordinateurs NECs PC-8801/9801
```

**Comment by OPNA2608** [comment]
Thanks for catching the extra dots, I only run a bare WM which doesn't utilize .desktop files!

**Comment by trebmuh** [comment]
Good catch @OPNA2608 :+1: 

That's what happen when you:
- try to package another software
- have a discussion about a different topic on IRC
- manage to fix a coat rack without having propers tools to use
- try to answer all and every questions of a 8yo niece
- think you should focus on only one thing at a time
- and because that's not enough, decide to contribute a 2-liners to BambooTracker

:smile:

**Comment by rerrahkr** [comment]
Thanks for your contributions!

**Pull request merged**